### PR TITLE
Fix some invalid doc yaml and improve docs

### PR DESCRIPTION
**Pull request by sean797**

### CODE REVIEW

File: modules/foreman_compute_resource.py [hunk 1]
```diff
@@ -23,11 +23,13 @@
...
 description:
 - Create and delete Foreman Compute Resources using Foreman API
+requirements:
+    - "nailgun >= 0.28.0"
 options:
   name:
     description: compute resource name
     required: true
-  description: compute resource description
+    description: compute resource description
```
Comment (akofink) [inline modules/foreman_compute_resource.py]: Where's the option name?

```
description:
  description: compute resource description
```

**Comment by sean797** [comment]
Updated, thanks @akofink 👍

**Comment by sean797** [comment]
I considered getting Travis to run `for file in `find modules/ -name *.py` ; do ansible-doc $(basename -s .py $file) ; done` or something similar but I think lets wait for https://github.com/theforeman/foreman-ansible-modules/pull/62

**Comment by akofink** [comment]
Yeah, I think that'd be a great idea.

**Pull request merged**